src/api.js

```diff
@@ -104,6 +104,19 @@ const endpoints = {
   },
   allVulnerabilitiesForRepo: (name) =>
     `/v2/_zot/ext/search?query={CVEListForImage(image: "${name}"){Tag Page {TotalCount ItemCount} CVEList {Id Title Description Severity Reference PackageList {Name InstalledVersion FixedVersion}}}}`,
+  cveDiffForImages: (minuend = {}, subtrahend = {}, { pageNumber = 1, pageSize = 3 }) => {
+    let imageInput = (img) => {
```

Comment:
if the value for the variable doesn't change, please use 'const' for declaration